src/azure/basic-azure-cli-facade.ts

```diff
@@ -76,7 +77,23 @@ export class BasicAzureCliFacade implements AzureCliFacade {
     const result = await this.shellRunner.run(
       `az tag list --subscription ${subscription.id}`,
     );
-    this.checkForErrors(result);
+
+    // we need to catch the case where certain subscriptions might not be accessible for the user.
+    // we will just ignore this in that case and continue.
+    try {
+      this.checkForErrors(result);
+    } catch (e) {
+      if (
+        e instanceof MeshAzurePlatformError &&
+        e.errorCode == AzureErrorCode.AZURE_UNAUTHORIZED
+      ) {
+        log.warning(
```

Comment:
I would actually propose to do this within the `this.checkForErrors` method with the following behavior:

Detect if missing a certain permission, if so, log a warning mentioning you do not have the correct permissions for executing command XYZ and log the `stderr` too so the user can see the exact error that was given.

Do make sure that it doesn't result in a false positive when not logged in though.

Reply:
hm that should work as the "not logged in" error is handled seperatly.